migrations/pom.xml

```diff
@@ -86,4 +86,36 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>bundleMigrationCmdTools</id>
```

Comment:
Added a profile that uses shade plugin for building migration command-line tools.

This is because `migrations` is both a dependency module and a set of [two] migration tools.

Reply:
But it's still the same artifact, right? When we release it and it goes on Maven Central, wouldn't we want to have them as separate jars? One being the library, another being the CLI tool(s)?

Reply:
Yeah. Should we have the cli tools in a separate module?
Most of the time we need just a library. A normal migration we need only for testing or for manual migration. Continuous migration is needed, hopefully, only once. I was thinking only to compile CLI tools only when we need them. Let me know if you want to have a module for CLI tools.

Reply:
Yeah, I think that's a good idea.
Thanks for already having done it 😄!